subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/isolated/IsolatedProjectsToolingApiIModelQueryIntegrationTest.groovy

```diff
@@ -94,6 +94,47 @@ class IsolatedProjectsToolingApiIModelQueryIntegrationTest extends AbstractIsola
         fixture.assertStateLoaded()
     }
 
+    def "can cache models for tasks"() {
```

Comment:
🤔 I would suggest providing this test with some explanation of what exactly happens and what would be the opposite of `can cache models for tasks`

Reply:
I've considered it's a good(enough) name because of API that tested here is like a 
`connection.model(type).forTasks(tasks)`